cfg/config.go

```diff
@@ -110,6 +115,43 @@ func applyEnvs(json *simplejson.Json) error {
 	return nil
 }
 
+func applyEnvsV2(json *simplejson.Json) error {
```

Comment:
Let's make a single function for env variables, not 2 different with different logic:
`applyEnvsV2` applies envs that starts with `$`
`applyEnvs` applies envs that starts with `FILE_D`